### PR TITLE
Added basic onboarding

### LINKED ISSUE #31: Create start screen
Create start screen with language selection and help information

### PR DESCRIPTION
**Pull request by kramarov-evg**
- Added preferences setup screen
- Added saving preferences to SharedPreferences
- Added semi-transparent guide to app
- Configured to show guide and pref-s setup only on first launch
- Some new icons

Closes #31

### CODE REVIEW

**Comment by kramarov-evg** [comment]
@LeonidVolohov, We'll need to discuss some things here

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt [hunk 1]
```diff
@@ -1,33 +1,116 @@
...
+        groupLanguageSelection = findViewById(R.id.language_selection_group)
+
+        if (firstLaunch) {
+            var currency = ""
+            var language = ""
+            Log.i("MAIN", "Wrote false to SharedPrefs")
+            Log.i(
+                "MAIN",
+                "Curr value in SharedPrefs: ${sharedPrefs.getBoolean("first_launch", true)}"
+            )
```
Comment (LeonidVolohov) [inline app/src/main/java/com/stonks/ui/main/MainActivity.kt]: Do we need this log in the future?

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -10,14 +10,169 @@
...
+            app:layout_constraintTop_toBottomOf="@id/language_selection_text"
+            android:checkedButton="@id/english_language">
+
+            <RadioButton
+                android:id="@+id/english_language"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/margin_padding_size_small"
+                android:drawableTop="@drawable/ic_united_states"
+                android:text="ENG" />
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -10,14 +10,169 @@
...
+
+            <RadioButton
+                android:id="@+id/russian_language"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="@dimen/margin_padding_size_small"
+                android:drawableTop="@drawable/ic_russia"
+                android:text="RU" />
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -10,14 +10,169 @@
...
+        </RadioGroup>
+
+        <Button
+            android:id="@+id/save_preferences_btn"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginEnd="16dp"
+            android:layout_marginBottom="16dp"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: There is margin size for it in dimes.xml

File: app/src/main/res/layout/activity_main.xml [hunk 5]
```diff
@@ -10,14 +10,169 @@
...
+            android:text="GO"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

File: app/src/main/res/layout/activity_main.xml [hunk 6]
```diff
@@ -10,14 +10,169 @@
...
+            app:layout_constraintStart_toStartOf="parent" />
+
+    </androidx.constraintlayout.widget.ConstraintLayout>
+
+    <RelativeLayout
+        android:id="@+id/guide_overlay_first_layout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_above="@id/bottom_navigation"
+        android:background="#CC666666"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/colors.xml

File: app/src/main/res/layout/activity_main.xml [hunk 7]
```diff
@@ -10,14 +10,169 @@
...
+            android:textAlignment="center"
+            android:textColor="?attr/colorAccent"
+            android:textSize="@dimen/text_size_xlarge" />
+
+        <TextView
+            android:id="@+id/first_guide_page_next_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_below="@id/first_guide_page_main_text"
+            android:text="(Tap to continue)"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

File: app/src/main/res/layout/activity_main.xml [hunk 8]
```diff
@@ -10,14 +10,169 @@
...
+            android:scaleType="fitXY"
+            android:src="@drawable/ic_guide_bottom_nav" />
+    </RelativeLayout>
+
+    <RelativeLayout
+        android:id="@+id/guide_overlay_second_layout"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_above="@id/bottom_navigation"
+        android:background="#CC666666"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/colors.xml

File: app/src/main/res/layout/activity_main.xml [hunk 9]
```diff
@@ -10,14 +10,169 @@
...
+            android:scaleType="fitXY"
+            android:src="@drawable/ic_arrows_to_currencies" />
+
+        <TextView
+            android:id="@+id/second_guide_page_main_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_above="@id/second_guide_page_next_text"
+            android:gravity="center_horizontal"
+            android:text="Choose base &amp; target"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

File: app/src/main/res/layout/activity_main.xml [hunk 10]
```diff
@@ -10,14 +10,169 @@
...
+            android:textColor="?attr/colorAccent"
+            android:textSize="@dimen/text_size_xlarge" />
+
+        <TextView
+            android:id="@+id/second_guide_page_next_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_alignParentBottom="true"
+            android:gravity="center_horizontal"
+            android:text="(Tap to continue)"
```
Comment (LeonidVolohov) [inline app/src/main/res/layout/activity_main.xml]: Take out to res/values/strings.xml

**Review by LeonidVolohov — COMMENTED** [review]
Done very nicely, but you need to make changes to xml files

File: app/src/main/java/com/stonks/ui/main/MainActivity.kt (hunk 1 above)
Comment (kramarov-evg) [reply app/src/main/java/com/stonks/ui/main/MainActivity.kt]: Of course no, once more forgotten logging

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 5 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 6 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 7 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 8 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 9 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

File: app/src/main/res/layout/activity_main.xml (hunk 10 above)
Comment (kramarov-evg) [reply app/src/main/res/layout/activity_main.xml]: DONE

**Review by LeonidVolohov — APPROVED** [review]

**Pull request merged**